cs-vmware/src/main/java/io/cloudslang/content/vmware/services/VmService.java

```diff
@@ -100,7 +100,7 @@
         ConnectionResources connectionResources = new ConnectionResources(httpInputs, vmInputs);
         try {
             ManagedObjectReference vmMor = new MorObjectHandler().getVmMor(connectionResources,
```

Comment:
duplicated 6 times in this class, this can also be found in GuestService class and in ClusterComputeResourceService class